app/assets/v2/js/pages/onboard.js

```diff
@@ -0,0 +1,164 @@
+var current = 0;
+var words = [];
+
+$('.js-select2').each(function() {
+  $(this).select2();
+});
+
+var showTab = function(num) {
```

Comment:
seems like global namespace pollution, maybe wrap in IIFE ?

Reply:
A named IIFE  (cause I'm invoking it in the file later again )? 
Question : 
Wouln't a named IIFE end up polluting the global namespace too ? (I've used it only when I've wanted immediate invocation.)

Reply:
Oh thank for the lesson :D
makes sense on the long run 
@kziemianek 
Does this look ok ? 
https://gist.github.com/thelostone-mc/51d9d649d8802a823051c45a4ed886b8